x-pack/plugins/observability_log_explorer/public/components/log_explorer_top_nav_menu.tsx

```diff
@@ -103,3 +195,24 @@ const DiscoverLink = React.memo(
     );
   }
 );
+
+const OnboardingLink = React.memo(({ services }: Pick<LogExplorerTopNavMenuProps, 'services'>) => {
+  const locator = services.share.url.locators.get<ObservabilityOnboardingLocatorParams>(
+    OBSERVABILITY_ONBOARDING_LOCATOR
+  );
+
+  const onboardingUrl = locator?.useUrl({});
+
+  return (
+    <EuiButton
+      href={onboardingUrl}
+      target="_blank"
```

Comment:
Doesn't this always open in a new tab now? Sorry for being unclear, but this would be the behavior we're aiming for with such buttons in Kibana when possible:

- clicking the button navigates to the on-boarding without a full page reload
- selecting "open in a new tab/window" on the button opens the on-boarding in a new tab/window

Reply:
Oh, I see what you mean! 
I just added a change to fix this link behavior and the discover one, thanks for clarifying this!